frontend/views/components/ProfileCard.vue

```diff
@@ -0,0 +1,243 @@
+<template lang='pug'>
+tooltip(
+  :direction='direction'
+  :manual='true'
+  ref='tooltip'
+  :opacity='1'
+)
+  slot
```

Comment:
The designs show an underline style when hovering the ProfileCard trigger (`slot`).

We have an accessibility problem here: We can't reach this component using the keyboard. I will ping @mmbotelho on Figma to double check the focus states and cc you.

The strategy to do an accessible "popover" menu is very similar to the one used for modals. (#778). I can help you if you need! Slack (browser) do it perfectly, is a great reference ✨ I'm okay if you want to do this in another PR as well.